tests/python/rest_api/test_tasks.py

```diff
@@ -1025,7 +1025,7 @@ def test_cannot_rename_label_to_duplicate_name(self, tasks, labels, admin_user):
         assert "All label names must be unique" in response.text
 
     def test_cannot_add_foreign_label(self, tasks, labels, admin_user):
-        task = list(tasks)[0]
+        task = [t for t in tasks if t["project_id"] is None][0]
```

Comment:
Why do we need this condition?

Reply:
We cannot add a label to a task that belongs to a project. This check avoids this.